sdk/cpp/core/src/service_provider.cpp

```diff
@@ -25,17 +25,12 @@
 //
 //////////////////////////////////////////////////////////////////
 
-#ifndef SERVICE_HPP
-#define SERVICE_HPP
-
-#include <string>
-#include <memory>
+#include "path_api.hpp"
 
 namespace ydk {
-class Service
-{
-};
 
+path::ServiceProvider::~ServiceProvider()
```

Comment:
Thanks, will move `ServiceProvider` out of `path` namespace, and address other comments in upcoming commit.